vortexasdk/api/voyages.py

```diff
@@ -189,4 +189,4 @@ class VoyageEnrichedItem(BaseModel):
             ]
         ]
     ] = None
-    latest_product_details: Optional[List[EntityWithSingleLayer]] = None
```

Comment:
What will happen to current SDK users when we release this fix?

Reply:
see my comment in the main thread